src/components/curriculum/CurriculumReactBootcamp.js

```diff
@@ -79,11 +82,12 @@ const CurriculumReactBootcamp = ({
       </Section>
       <Section
         {...commonProps}
-        title="React Bootcamp Day 2"
+        title={`Day 2 (${trainingDateByDay({
+          training,
+          day: 2,
+        })} ${trainingTime})`}
```

Comment:
@ric9176 why don't we pass the trainingTime as a separate prop, so the Section can decide how to render it internally.